### PR TITLE
Fix for messages notice & terminology in search form

### PR DESCRIPTION
**Pull request by rgilman**

### CODE REVIEW

File: buddypress/common/js-templates/messages/parts/bp-messages-form.php [hunk 1]
```diff
@@ -53,13 +53,13 @@ class="send-to-input"
 	<div id="bp-message-content"></div>
 
 	<?php
-	if ( 'mac' === $os ) {
+	if ( 'window' === $os ) {
```
Comment (iangilman) [inline buddypress/common/js-templates/messages/parts/bp-messages-form.php:56]: I'm not familiar with the code, so I don't know, but is it supposed to be window, not windows?

File: buddypress/common/js-templates/messages/parts/bp-messages-form.php [hunk 2]
```diff
@@ -53,13 +53,13 @@ class="send-to-input"
...
 		?>
-		<p class="bp-message-content_foot_note"><span class="space_note">Please note:</span><span class="space_note"><strong><?php esc_html_e( 'Shift+Return', 'buddyboss' ); ?> </strong> <?php esc_html_e( 'to add a new line', 'buddyboss' ); ?></span><strong><?php esc_html_e( 'Return', 'buddyboss' ); ?></strong><?php esc_html_e( ' to Send', 'buddyboss' ); ?></p>
+		<p class="bp-message-content_foot_note"><span class="space_note">Please note:</span><span class="space_note"><strong><?php esc_html_e( 'Shift+Enter', 'buddyboss' ); ?> </strong> <?php esc_html_e( 'to add a new line', 'buddyboss' ); ?></span><strong><?php esc_html_e( 'Enter', 'buddyboss' ); ?></strong><?php esc_html_e( ' to Send', 'buddyboss' ); ?></p>
 		<?php
-	} elseif ( 'window' === $os ) {
+	} else {
```
Comment (iangilman) [inline buddypress/common/js-templates/messages/parts/bp-messages-form.php:60]: It looks like this change makes it so Linux will now use the "return" terminology… Is that desired? I would have thought they would be using "enter".

File: buddypress/common/js-templates/messages/parts/bp-messages-form.php (hunk 1 above)
Comment (rgilman) [reply buddypress/common/js-templates/messages/parts/bp-messages-form.php:56]: Thanks for checking this important level of detail! Surprisingly enough, 'window' is correct.

File: buddypress/common/js-templates/messages/parts/bp-messages-form.php (hunk 2 above)
Comment (rgilman) [reply buddypress/common/js-templates/messages/parts/bp-messages-form.php:60]: You may well be right. I'm hoping San can fill us in. Easy to fix once we know.

File: buddypress/common/js-templates/messages/parts/bp-messages-form.php (hunk 1 above)
Comment (iangilman) [reply buddypress/common/js-templates/messages/parts/bp-messages-form.php:56]: Good to know!

**Pull request merged**